### PR TITLE
Transition to Python 3.8

### PR DESCRIPTION
**Pull request by t0mpr1c3**
Also requires

PyQt5~=5.12
PyInstaller~=4.1

and a suitably patched version of fbs.

### CODE REVIEW

**Comment by t0mpr1c3** [comment]
Needs to be merged at the same time as https://github.com/AllYarnsAreBeautiful/fbs/pull/3

File: requirements.txt [hunk 1]
```diff
@@ -1,13 +1,14 @@
-fbs~=1.2
-PyInstaller~=3.4
-pyqt5==5.9.2
+#fbs~=1.2
+fbs @ git+https://github.com/t0mpr1c3/fbs@py38
```
Comment (dl1com) [inline requirements.txt:2]: We should choose a sensible tag name in the ayab fbs fork repo to identify the version and use it here to specify the version

File: requirements.txt [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+PyInstaller~=4.1
+pyqt5~=5.12
 Pillow==7.1.2
 pyserial~=3.4
 sliplib==0.4.0
 pytest==5.4.1
 mock==3.0.5
 bitarray-hardbyte==1.1.0
-simpleaudio==1.0.2
+simpleaudio==1.0.4
```
Comment (dl1com) [inline requirements.txt:11]: Intended change?

**Comment by dl1com** [comment]
https://github.com/AllYarnsAreBeautiful/ayab-desktop/blob/1.0.0-dev/README.md needs an update as well, 3.6 is mentioned several times.

File: requirements.txt (hunk 1 above)
Comment (t0mpr1c3) [reply requirements.txt:2]: Yes

File: requirements.txt (hunk 2 above)
Comment (t0mpr1c3) [reply requirements.txt:11]: Yes

File: requirements.txt [hunk 3]
```diff
@@ -1,13 +1,14 @@
-fbs~=1.2
-PyInstaller~=3.4
-pyqt5==5.9.2
+#fbs~=1.2
+fbs @ git+https://github.com/t0mpr1c3/fbs@py38
+PyInstaller~=4.1
```
[the remark this answers is not in the document]
Comment (t0mpr1c3) [reply requirements.txt:3]: We can probably delete those requirements

File: requirements.txt (hunk 1 above)
Comment (t0mpr1c3) [reply requirements.txt:2]: Let's use the tag 'ayab-py38'

**Pull request merged**